src/hardware/serialport/serialport.cpp

```diff
@@ -1337,12 +1339,21 @@ class SERIALPORTS final : public Module_base {
 				serialports[i] = new CNullModem (i, &cmd);
 				serialports[i]->serialType = SERIAL_PORT_TYPE::NULL_MODEM;
 				cmd.GetStringRemain(serialports[i]->commandLineString);
-				if (!serialports[i]->InstallationSuccessful)  {
+				if (!serialports[i]->InstallationSuccessful) {
 					delete serialports[i];
 					serialports[i] = NULL;
 				}
 			}
 #endif
+			else if(type=="serialmouse") {
+				serialports[i] = new CSerialMouse (i, &cmd);
+				serialports[i]->serialType = SERIAL_PORT_TYPE::MOUSE;
+				cmd.GetStringRemain(serialports[i]->commandLineString);
+				if (!serialports[i]->InstallationSuccessful) {
+					delete serialports[i];
+					serialports[i] = NULL;
```

Comment:
Suggest swapping in `nullptr` when touching existing code (small uplift!)

Reply:
Done.